test/units/module_utils/common/warnings/test_deprecate.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# (c) 2019 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import pytest
+
+import ansible.module_utils.common.warnings as warnings
+
+from ansible.module_utils.common.warnings import deprecate
+
+
+def test_deprecate_message_only():
+    deprecate('Deprecation message')
+    warnings._global_deprecations == [{'msg': 'Deprecation message', 'version': None}]
```

Comment:
```suggestion
    assert warnings._global_deprecations == [{'msg': 'Deprecation message', 'version': None}]
```

Reply:
That is covered in `test_deprecate_with_version()`

Reply:
Good catch!